components/raftstore/src/store/worker/region.rs

```diff
@@ -258,6 +258,7 @@ where
         notifier: SyncSender<RaftSnapshot>,
         for_balance: bool,
         allow_multi_files_snapshot: bool,
+        start: UnixSecs,
```

Comment:
How about put `start` as member of `SnapGenContext` ? Seems it's reasonable as a kind of "context" of snapshot generation. And then we don't need to pass the variable all the way along.

Besides, why not use `Instant` type ? `Instant` is monotonic, and we use it in most conditions in TiKV.

Reply:
> How about put start as member of `SnapGenContext` ?
yes，It is better to put `start` into the context .
 >why not use Instant type?
 The start only record the snapshot's beginning timestamp, and it will be sent into another thread to record the total duration of snapshot.